scripts/scripts.js

```diff
@@ -42,13 +42,26 @@ async function loadFonts() {
   }
 }
 
+function autolinkModals(main) {
+  main.addEventListener('click', async (e) => {
```

Comment:
i think it's possible that modals could be in the header or footer. 
```suggestion
  document.body.addEventListener('click', async (e) => {
```

Reply:
as decorateMain() can run multiple times, `document.body.addEventListener` would add multiple listeners